R/plot_shinygosling.R

```diff
@@ -68,22 +70,21 @@ plot_granges <-
     categories <- c("likely_benign", "ambiguous", "likely_pathogenic")
     colormapping <- c("#89d5f5", "gray", "#f56c6c")
 
-    # Get range from GRanges object
+    ## Get range from GRanges object
     r <- range(gr)
 
-    # This fixes the bug if .gosling directory does not already exist
-    if (!dir.exists(".gosling")){
-        dir.create(".gosling")
-    }
+    ## This fixes the bug if .gosling directory does not already exist
+    gosling_cache <- file.path(R_user_dir("AlphaMissenseR", which = "cache"), ".gosling")
```

Comment:
Please make sure that lines do not exceed 80 characters, following formatting conventions in the package, e.g.,
```
    gosling_cache <- file.path(
        R_user_dir("AlphaMissenseR", which = "cache"),
        ".gosling"
    )
```

Reply:
Outdated, see lines 76.